packages/document/main-doc/docs/en/configure/app/source/design-system.mdx

```diff
@@ -851,19 +851,19 @@ const designSystem = {
 };
 ```
 
-You can then style the element differently in dark mode using the `dark:` prefix:
+Then, you can use the `dark:` prefix to set different styles for elements in dark mode:
 
 ```html
 <div class="text-gray-700 dark:text-gray-200">
   {/* ...  */}
 </div>
 ```
 
-Note that since these `screen variants` are implemented in `Tailwind CSS`, ** you cannot use this method to combine breakpoints with dark modes **, for example `md:dark:text-gray-300` will not work.
+Please note that since these screen variants are implemented in `Tailwind CSS`, **it is not possible to use this method to combine breakpoints with dark mode**, e.g. `md:dark:text-gray-300` will not work.
```

Comment:
```suggestion
Please note that since these screen variants are implemented in TailwindCSS, **it is not possible to use this method to combine breakpoints with dark mode**, e.g. `md:dark:text-gray-300` will not work.
```